pkg/streamingpromql/operators/functions/range_vectors_test.go

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/prometheus/prometheus/blob/main/promql/functions.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Prometheus Authors
+
+package functions
+
+import (
+	"testing"
+
+	"github.com/prometheus/prometheus/promql"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_runDoubleExponentialSmoothing(t *testing.T) {
+	type args struct {
```

Comment:
[nit] Why not just inline these fields into the struct below?

Reply:
Make it little clearer which one is arguments and which one are `want`s. I think I will just keep it this way.